cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -77,7 +77,7 @@ func NewCMServer() *CMServer {
 		NodeMilliCPU:            1000,
 		NodeMemory:              resource.MustParse("3Gi"),
 		SyncNodeList:            true,
-		SyncNodeStatus:          false,
+		SyncNodeStatus:          true,
```

Comment:
See https://github.com/GoogleCloudPlatform/kubernetes/pull/5399, which should address you concern.  Enabling sync status will mess up the cluster, since it operates in two different modes (node controller pull status vs kubelet push status)